workers/setup.cfg

```diff
@@ -2,15 +2,17 @@
 # Read more about `setup.cfg`:
 # https://docs.python.org/3/distutils/configfile.html
 
-
 [flake8]
 # W503: https://github.com/python/black#line-breaks--binary-operators
 # E203: https://github.com/psf/black#slices
 # These settings are compatible with Black
 ignore = W503, E203
 inline-quotes = double
 max-line-length = 88
-
+exclude =
+per-file-ignores =
+    ohsome_quality_analyst/indicators/mapping_saturation/*:N8
```

Comment:
Why should we ignore all naming rules for the mapping saturation implementation?

Reply:
If the intention was to ignore existing errors, please add a TODO note and apply the rules to all failing files explicitly (not with a star). This ensures that we explicitly know which files should be fixed, soon.

Reply:
Because there are many. This should probably be adressed seperatly, should it not?